tests/block-period-api.test.js

```
@@ -0,0 +1,317 @@
+      () =>
+        BlockPeriodService.getAvailableBlockPeriods(
+          "tenant-1",
+          "camping-a",
+          "not-a-date",
+          "2026-06-30",
+          1,
+          null,
+        ),
+      (error) =>
+        error instanceof BadRequestError &&
+        error.code === "invalid_date_range",
```

ESLint

[error] Delete `⏎·······`

(prettier/prettier)